draft-ietf-quic-transport.md

```diff
@@ -5654,7 +5663,7 @@ RETIRE_CONNECTION_ID frames contain the following fields:
 Sequence Number:
 
 : The sequence number of the connection ID being retired.  See
-  {{retiring-cids}}.
+  {{retire-cid}}.
```

Comment:
I can't suggest here (damn GitHub).  But what you want is a semi-colon: '; see {{retire-cid}}.'

Reply:
Done in 3cf8eaa